MinionsSampleTests/MinionRepoTests.swift

```diff
@@ -7,13 +7,13 @@ class MinionRepoTests: XCTestCase {
 
     func testGetMinionsAsynchronouslyParsesData() {
         let fakeJsonParser = FakeJsonParser()
-        let minionRepo = MinionsRepo(parser: fakeJsonParser)
```

Comment:
There's a possibility that I've misinterpreted the function of this test, but I'll check over it again later.